Aztec/Classes/Public/Extensions/NSAttributedString+Attachments.swift

```diff
@@ -22,4 +22,26 @@ extension NSAttributedString
             }
         }
     }
+
+    /// Determine the character ranges for an attachment
+    public func ranges(forAttachment attachment: NSTextAttachment) -> [NSRange]
+    {
+        // find character range for this attachment
+        let range = NSRange(location: 0, length: length)
+
+        var refreshRanges = [NSRange]()
```

Comment:
I think we should rename this variable, since this method is not really tied to refreshing.